### DISCUSSION TITLE
Log4j / Log4Shell

### DISCUSSION THREAD

**Discussion by BrendanxP** [FaserF/hassio-addons#51] [General]
Just to make sure, these apache add-ons are not vulnerable to the latest exploit, right?

**Comment by FaserF** [comment]
As far as I understand the exploit to this addon is not vulnerable. 
Java systems are affected and this addon doesnt use java and doesnt have liblog4j2-java installed.

If someone has forked this addon and installed liblog4j2-java somehow, my recommendation is to update it to the latest release which contains a fix against it.

**Reply by BrendanxP** [reply]
Thanks a lot for your confirmation! The add-on has been working great for quite a long time :)
